node-src/tasks/upload.ts

```diff
@@ -119,10 +119,10 @@ export const validateFiles = async (ctx: Context) => {
 export const traceChangedFiles = async (ctx: Context, task: Task) => {
   if (!ctx.turboSnap || ctx.turboSnap.unavailable) return;
   if (!ctx.git.changedFiles) return;
-  if (!ctx.fileInfo.statsPath) {
+  if (!ctx.fileInfo?.statsPath) {
     // If we don't know the SB version, we should assume we don't support `--stats-json`
     const nonLegacyStatsSupported =
-      ctx.storybook?.version && semver.gte(semver.coerce(ctx.storybook.version), '8.0.0');
+      ctx.storybook?.version && semver.gte(semver.coerce(ctx.storybook.version) || '', '8.0.0');
```

Comment:
Same here
```suggestion
      ctx.storybook?.version && semver.gte(semver.coerce(ctx.storybook.version) || '0.0.0', '8.0.0');
```

Reply:
Yeah that's a good call. I'll make that change!